widget/widgetAssets/locales/en.json

```diff
@@ -22,6 +22,8 @@
         "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_DROP_A_FILE":"Drop a file",
         "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_USE_STANDARD_UPLOADER":"or click to use standard uploader",
         "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_FILE_SIZE_SHOULD_NOT_EXCEED_25MB":"File Size should not exceed 25MB",
```

Comment:
EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_DROP_A_FILE --> BULK_IMPORT_DROP_A_FILE_TITLE
"Drop a file" --> "Drop a File"
BULK_IMPORT_USE_STANDARD_UPLOADER_TITLE":"or Click to Use Standard Uploader
File Size should not exceed 25MB" ---> File Size Should Not Exceed 25MB"

Reply:
DONE